src/Tizen.NUI/src/internal/Registry.cs

```diff
@@ -77,8 +76,7 @@ internal static void Register(BaseHandle baseHandle)
         /// <param name="baseHandle"> The instance of BaseHandle (C# base class)</param>
         internal static void Unregister(BaseHandle baseHandle)
         {
-            RefObject refObj = baseHandle.GetObjectPtr();
```

Comment:
How about deleting 'GetObjectPtr' that has a memory leak issue?
If this api is an internal api that is not used.
If the developer does not know and uses it, the memory leak issue may occur, so if it is difficult to delete it, it would be better to leave a comment.

Reply:
it is removed.